azure/index.d.ts

```diff
@@ -418,3 +418,14 @@ export function createAzureSubscriptionClient<T>(context: AzExtClientContext, cl
 export declare namespace uiUtils {
     export function listAllIterator<T>(iterator: PagedAsyncIterableIterator<T>): Promise<T[]>
 }
+
+export declare class ConfirmationStep<T extends IActionContext> extends AzureWizardPromptStep<T> {
```

Comment:
IMO this should go into the `utils` package. This could be useful to any extension using the wizard framework, irrespective of Azure.

Reply:
We could do a generic `ConfirmationStep` that has "Yes" and "No" options, along with a `DeleteConfirmationStep` with a "Delete" and "Cancel" or whatever the preferred two buttons are.

Reply:
👍 moved them into utils